src/merkle.rs

```diff
@@ -1810,6 +1810,37 @@ impl Element for [u8; 32] {
     }
 }
 
+pub fn get_merkle_tree_len_generic<
```

Comment:
Some documentation would be good as it is quite interesting in which cases which values are returned.

Reply:
Added missing explanation